config/src/config/mod.rs

```diff
@@ -278,7 +290,9 @@ impl NodeConfig {
             );
 
             if self.validator_network.is_none() {
-                self.validator_network = Some(NetworkConfig::default());
+                let mut network_config = NetworkConfig::default();
+                network_config.network_id = NetworkId::validator_network();
```

Comment:
maybe we should make a new function that takes in a network id? maybe not...

Reply:
I can take a look into that, there's just a lot of defaults used, and I think when I started this I wasn't sure if the higher level configs call NetworkConfig default.

Reply:
I've combined these.  But, can't fully get rid of `NetworkConfig::default()` because it is used in deserialization.